pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -100383,7 +100383,6 @@ self: {
        license = stdenv.lib.licenses.asl20;
        hydraPlatforms = stdenv.lib.platforms.none;
        maintainers = with stdenv.lib.maintainers; [ maralorn ];
-       broken = true;
```

Comment:
I think you'll have to remove `ghcide` from the list of `broken-packages` in `configuration-hackage2nix.yaml` instead of editing it directly in this file.

Reply:
ghcide is not under broken-packages tho, I don't really know how it's marked broken :shrug:. it's really weird.

Reply:
Huh, that's strange.

Maybe @maralorn knows what is going on here?

Reply:
Funfact: We couldn‘t even enter ghcide into the list of broken packages, because I am registered as its maintainer and the hackage2nix doesn‘t let you break maintained packages.

The issue is, that the last hackage-revision of ghcide introduced a test dependency on a package which does not exist on hackage. You can see, that cabal2nix passes it to the derivation with a `{ .. = null; }` and then marks the package as broken. But we have ghcide tests disabled anyways, so that it would actually build …

Reply:
shake-bench is used only for benchmarking iirc